src/js/components/Tip/Tip.js

```diff
@@ -0,0 +1,88 @@
+import React, {
+  Children,
+  cloneElement,
+  forwardRef,
+  useContext,
+  useState,
+} from 'react';
+import { ThemeContext } from 'styled-components';
+
+import { Box } from '../Box';
+import { Drop } from '../Drop';
+import { useForwardedRef } from '../../utils/refs';
+
+const Tip = forwardRef(({ children, content, dropProps }, tipRef) => {
+  const theme = useContext(ThemeContext);
+  const [over, setOver] = useState(false);
+
+  const componentRef = useForwardedRef(tipRef);
+
+  // In cases the child is a primitive
+  const wrapInvalidElement = () => {
+    // Handle the use case of a primitive string child
+    // so we'll be able to assign ref and events on the child.
+    return !React.isValidElement(children) ? <span>{children}</span> : children;
+  };
+
+  /* Three major use case 
+    1. Tip has a single child + it is a React Element => Great!
+    2. Tip has a single child +  not React Element => span to wrap the child.
+    3. Tip has more than one child => Abort, display Children.only error 
+  */
+  const child =
```

Comment:
I think this can be simplified to `const child = wrapInvalidElement(Children.only(children));`

Reply:
It would be cleaner, wouldn't it 😉 
`Children.only(children)` is failing when the children are not valid to React element, so it is required to wrap the child to be a React Element (i.e. `warpInvalidElement`) before calling `Children.only(children)`.